### DISCUSSION TITLE
Get back old bookmark UI

### DISCUSSION THREAD

**Discussion by coding-sunshine** [dailydotdev/daily#638] [Feature Requests]
With the new UI, we have to go to the individual link menu and click on the bookmark. Before this, it was just simply available on the main link UI. that was a better UX I think.

**Comment by idoshamun** [comment]
Hi @coding-sunshine,

Thanks for sharing your thoughts. We are trying different approaches for the interaction buttons on the card due to several requests we received in the past. It's still in the experimentation phase and not the final rollout. Stay tuned :)

**Comment by nrndev** [comment]
@idoshamun would love to know if there is any progress/development on the bookmark icon on the card itself rather than clicking via the options menu

**Reply by idoshamun** [reply]
Hi @narenarjun,

We decided to proceed with adding the bookmarks to the share context menu. See below (it's already live):
<img width="357" alt="image" src="https://user-images.githubusercontent.com/1993245/205849303-2a9b4d37-50fe-43ea-b290-18a86ff28459.png">
This approach serves both purposes of share and bookmark. I hope this makes sense to you 🙏

**Comment by ricardogouveia3** [comment]
I wasn't aware of this bookmark feature, and I just came here today for the first time on the feature suggestions to exactly suggest it. It is not obvious to find a 'save' button along the share option. Twitter does something like it, and it's terrible. I do believe you could have 4 icons on that footer menu to the article, or at least have the option named 'Share & Save' with a new icon.

It would be nice to have also, on the articles grid, a featured line with bookmarked unread articles, to give the user a chance to remember to read those and avoid the bookmark list to become an endless void of unread articles. The 'mark as read' is already done I guess.

**Comment by nimrodkra** [comment]
@coding-sunshine @ricardogouveia3 @narenarjun we brought back the bookmark button to the card - enjoy! Thanks for the feedback and for the patience 🙏

![IMG_8444](https://github.com/dailydotdev/daily/assets/41470823/3d86b8d1-0d7c-44c4-a1c7-052a650696fd)
